docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -72,20 +72,30 @@ or for a database-role combination.
 
 #### Enable on a database
 
+:::important
+If you use PostgreSQL 14 or earlier, [upgrade](/docs/products/postgresql/howto/upgrade) to
+PostgreSQL 15 or later to enable `pgaudit` on your database.
+:::
+
 1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
 1. Run the following query:
 
    ```sql
-   SELECT aiven_extras.set_pgaudit_parameter('log', DATABASE_NAME, 'ddl');
+   SET pgaudit.feature_enabled='all';
```

Comment:
This should be:
```
SET pgaudit.log='ddl';
```

`feature_enabled` is not a pgaudit GUC, its just a flag from user_config